packages/design/src/icons/Icon.css

```diff
@@ -67,28 +71,26 @@
 }
 
 .badInvoice {
-  fill: var(--color-red);
+  fill: var(--color-critical);
 }
 
 .payment,
 .expense {
   fill: var(--color-orange);
 }
 
-.edit {
-  fill: var(--color-brown);
-}
-
 .archive {
   fill: var(--color-blue);
 }
 
-.excel {
-  fill: rgb(31, 115, 71);
+.video {
+  fill: var(--color-lightBlue);
 }
 
-.file {
-  fill: var(--color-greyBlue);
+/* non-Atlantis colors to match brand colors of these file-types */
+
```

Comment:
Super small nitpick but I think the line between the comment and the CSS class should be removed

Reply:
Is that a thing? I thought best practice was to add lines between for readability?

Reply:
@chris-at-jobber nevermind. I looked at examples and only found one that followed the way I suggested.